scripts/abundance-dist.py

```diff
@@ -87,7 +87,9 @@ def main():
     print('HT sizes:', hashsizes, file=sys.stderr)
     print('outputting to', args.output_histogram_filename, file=sys.stderr)
 
-    if os.path.exists(args.output_histogram_filename):
+    if args.output_histogram_filename in ('-', '/dev/stdin'):
+        pass
+    elif os.path.exists(args.output_histogram_filename):
```

Comment:
Ah, but here the logic is going to exit *because* /dev/stdin already exists. So I'll leave this one in.